src/tests/ftest/server/daos_server_dump.py

```diff
@@ -25,43 +26,109 @@ def __init__(self, *args, **kwargs):
         self.start_servers_once = False
         self.setup_start_agents = False
 
+    def tearDown(self):
+        """Tear down after each test case."""
+        super().tearDown()
+
+        # force test status !!
+        # use mangling trick described at
+        # https://stackoverflow.com/questions/3385317/private-variables-and-methods-in-python
+        # to do so
+        self._Test__status = 'PASS'
```

Comment:
(pylint-attribute-defined-outside-init) Attribute '_Test__status' defined outside __init__

Reply:
Given this is all internal test code anyway and we're not trying to publish a supported API why is there a need to use double underscore, when one or even none would suffice.